Objects/Objects/BuiltElements/Rebar.cs

```diff
@@ -20,6 +159,17 @@ public class Rebar : Base, IHasVolume, IDisplayValue<List<Mesh>>
 
 namespace Objects.BuiltElements.TeklaStructures
```

Comment:
I think it's worth moving these classes to their own files, but.. I appreciate that begs the question, where should they live...  let's chat, maybe we get Alan and Jedd involved in that chat also...

Reply:
Sure! This is currently how we've been organizing our applciation-specific namespaces for application-specific subclasses, but open for reconsideration!